src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -50,7 +52,8 @@ public void baseClassSetup() {
 
     protected void prepareTestData() {
         dataBundle = getTypicalDataBundle();
-        removeAndRestoreTypicalDataBundle();
```

Comment:
On second thoughts, it'd be better to move the reference to `BaseComponentTestCase`, as `BaseComponentTestCase` already has `getTypicalDataBundle()` and `removeAndRestoreDataBundle(dataBundle)` in `removeAndRestoreTypicalDataBundle`.